src/applications/caregivers/tests/unit/components/FormDescriptions/AddressCountyDescriptions.unit.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import {
+  CaregiverCountyDescription,
+  VeteranCountyDescription,
+} from '../../../../components/FormDescriptions/AddressCountyDescriptions';
+
+describe('CG <CaregiverCountyDescription>', () => {
+  it('should render `va-additional-info` with status of `error`', () => {
```

Comment:
This description doesn't match the test. I it needs to validate the error status?

Reply:
Whoops... only updated half the description.